packages/coinjoin/src/client/CoinjoinPrison.ts

```diff
@@ -0,0 +1,56 @@
+export interface PrisonInmate {
+    id: string; // AccountUtxo/Alice.outpoint or AccountAddress scriptPubKey
+    sentenceStart: number;
+    sentenceEnd: number;
+    reason?: string;
+    roundId?: string;
+}
+
+export interface PrisonOptions {
+    roundId?: string;
+    reason?: string;
+    sentenceEnd?: number;
+}
+
+// Errored or currently registered inputs and addresses are sent here
+// inspiration: WalletWasabi/WabiSabi/Backend/Banning/Prison.cs
+
+export class CoinjoinPrison {
+    lastChange: number;
+    inmates: PrisonInmate[] = [];
+
+    constructor() {
+        this.lastChange = Date.now();
```

Comment:
not used

Reply:
[5239ec1](https://github.com/trezor/trezor-suite/pull/6712/commits/5239ec1cee6e303565e09581d67ece666cff2b0a)